pkgs/development/python-modules/python-youtube/default.nix

```diff
@@ -0,0 +1,78 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, poetry-core
+, dataclasses-json
+, isodate
+, requests
+, requests-oauthlib
+, pytestCheckHook
+, responses
+, pytest
+, pytest-cov
+,
+}:
+buildPythonPackage rec {
+  pname = "python-youtube";
+  version = "0.9.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "sns-sdks";
+    repo = "python-youtube";
+    rev = "v${version}";
+    hash = "sha256-uimipYgf8nfYd1J/K6CStbzIkQiRSosu7/yOfgXYCks=";
+  };
+
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace "poetry.masonry.api" "poetry.core.masonry.api"
+  '';
+
+  nativeBuildInputs = [
+    poetry-core
+  ];
+
+  propagatedBuildInputs = [
+    dataclasses-json
+    isodate
+    requests
+    requests-oauthlib
+  ];
+
+  pythonImportsCheck = [ "pyyoutube" ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    responses
+    pytest
+    pytest-cov
+  ];
+
+  disabledTests = [
+    # On both tests, upstream compares a string to an integer
+
+    /*
+      python3.10-python-youtube> >       self.assertEqual(m.viewCount, "160361638")
```

Comment:
We don't need to full details

Would be nice if this could quickly be fixed upstream

Reply:
https://github.com/sns-sdks/python-youtube/pull/160